hibernate-core/src/main/java/org/hibernate/bytecode/enhance/internal/bytebuddy/EnhancerImpl.java

```diff
@@ -167,18 +187,18 @@ private TypePool buildTypePool(final ByteBuddyEnhancementContext enhancementCont
 									.annotateField( AnnotationDescription.Builder.ofType( Transient.class ).build() )
 							.defineMethod( EnhancerConstants.TRACKER_CHANGER_NAME, void.class, Visibility.PUBLIC )
 									.withParameters( String.class )
-									.intercept( Advice.to( CodeTemplates.TrackChange.class ).wrap( StubMethod.INSTANCE ) )
+									.intercept( adviceTrackChange.wrap( StubMethod.INSTANCE ) )
```

Comment:
Wondering if it would make sense to also cache the `wrap()` call. It would avoid creating a new object each time.

Reply:
I was afraid you'd say that :)

On it..